boot/zephyr/boards/nrf52840_pca20035.conf

```diff
@@ -0,0 +1,39 @@
+# Enable RTT logging
```

Comment:
I'm confused. The next line (`CONFIG_LOG=n`) isn't enabling logging; it's disabling logging.